README.rst

```diff
@@ -17,46 +17,49 @@ Pytest container
-container images leveraging `testinfra
-<https://testinfra.readthedocs.io/en/latest/>`_. Assuming you want to
-automatically spin up a container for a test, then the `container` fixture will
-do exactly that (plus it will cleanup after itself):
+``pytest_container`` is a small and simple `pytest <https://pytest.org>`_ plugin
+to test container images via pytest fixtures and `testinfra
+<https://testinfra.readthedocs.io/en/latest/>`_. It takes care of all the boring
+tasks, like spinning up containers, finding free ports and cleaning up after
+tests, and allows you to focus on implementing the actual tests.
+
+Some of the features include:
+- automatically pull, launch & stop containers
+- build containers via a :file:`Dockerfile`
+- wait for containers to become healthy before executing tests
+- bind exposed container ports to free ports on the host
+- mount volumes via temporary directories
+- parallel test execution via pytest-xdist
+- dependency between container images
+- run the same test on as many container images as you'd like
+
+At its heart, ``pytest_container`` provides four fixtures to help you achieve
+everything that you need for your container testing. Spinning up a container
+image can be as simple as instantiating a
+:py:class:`~pytest_container.container.Container` and parametrizing a test
+function with the ``container`` fixture:
 
 .. code-block:: python
 
-   LEAP = Container(url="registry.opensuse.org/opensuse/leap:latest")
+   TW = Container(url="registry.opensuse.org/opensuse/tumbleweed:latest")
 
-   @pytest.mark.parametrize("container", [LEAP], indirect=["container"])
-   def test_leap(container):
+   @pytest.mark.parametrize("container", [TW], indirect=["container"])
+   def test_etc_os_release_present(container: ContainerData):
        assert container.connection.file("/etc/os-release").exists
 
 
-In the above example we created a
-:py:class:`pytest_container.container.Container` object which is just a
-container that is directly pulled from `registry.opensuse.org
-<https://registry.opensuse.org/>`_. The `container` fixture then receives this
-container via pytest's parametrization and returns a
-:py:class:`pytest_container.container.ContainerData` to the test function. In
-the test function itself, we can leverage testinfra to run some basic tests
-inside the container itself, e.g. check whether files are there, packages are
-installed, etc.pp.
+The fixture will automatically pull and spin up the container for you, stop it
+and remove it after the test ran. Your test function will receive an instance of
+:py:class:`pytest_container.container.ContainerData`, which includes besides the
+container's id and the class instance which created it also a `testinfra
+<https://testinfra.readthedocs.io/en/latest/>`_ connection. This connection can
+be used to run some basic tests inside the container itself, e.g. check whether
+files are there, packages are installed, etc.pp.
```

Comment:
```suggestion
The fixture automatically pulls and spins up the container, stops it
and removes it after the test is completed. Your test function receives an instance of
:py:class:`pytest_container.container.ContainerData` that includes besides the
container's id and the class instance which created it also a `testinfra
<https://testinfra.readthedocs.io/en/latest/>`_ connection. This connection can
be used to run basic tests inside the container itself. For example, you can check whether
files are present, packages are installed, etc.
```

> Your test function receives an instance of
:py:class:`pytest_container.container.ContainerData` that includes besides the
container's id and the class instance which created it also a `testinfra
<https://testinfra.readthedocs.io/en/latest/>`_ connection

I don't understand this sentence.